app/mailers/user_mailer.rb

```diff
@@ -16,4 +17,10 @@ def account_approved(user)
 
     mail(to: @user.email, subject: t('mailers.user_mailer.account_approved.subject'))
   end
+
+  def yesterday_modifications(user, yesterday_modifications)
```

Comment:
I would have named this `daily_newsletter` or `daily_changes_update`. I would prefer the word `changes` rather than `modifications`.